docker/prod/setup/preinstall-common.sh

```diff
@@ -52,6 +52,18 @@ apt-get install -yq --no-install-recommends \
 	libclang-dev \
 	git
 
+# Use jemalloc as memory allocator
+if [ "$USE_JEMALLOC" = "true" ]; then
+	apt-get install -yq libjemalloc2
+	if [ "$ARCHITECTURE" = "x64" ]; then
+		LIB_ARCH="x86_64"
+	elif [ "$ARCHITECTURE" = "arm64" ]; then
+		LIB_ARCH="aarch64"
+	elif [ "$ARCHITECTURE" = "ppc64le" ]; then
+		LIB_ARCH="powerpc64le"
+	fi
+	export LD_PRELOAD=/usr/lib/${LIB_ARCH}-linux-gnu/libjemalloc.so.2
```

Comment:
@top4ek aren't we supposed to also set that env variable when launching a container from this image?

Reply:
Hmm, looks like you're right. Just looked into fresh instance with this patch and it doesn't work. `export` is not set on build stage.

Reply:
I think for now there would need to be an env variable basically saying "launch with jemalloc" (can reuse `USE_JEMALLOC` for that) that sets the `LD_PRELOAD` path in the entrypoint.

Reply:
I hoped to bake LD_PRELOAD straight into the image on build stage to make it more "systemwide". Quick googling said that i can't do that inside dockerfile's RUN.

Reply:
You can use the existing entrypoint.sh file, and simply export the relevant variables if `USE_JEMALLOC` env variable is set to `true` by the user?

Reply:
Sure, but `entrypoint.sh` is for only all-in-one target, so I'm going to create additional entrypoint script for slim target and don't want to copy-paste same code in both scripts.